lib/linguist/languages.yml

```diff
@@ -4056,6 +4078,7 @@ Scheme:
   color: "#1e4aec"
   extensions:
   - ".scm"
+  - ".sch"
```

Comment:
The extent of my Estonian knowledge in one picture:

<img width="528" alt="screen shot 2017-07-28 at 6 01 06 pm" src="https://user-images.githubusercontent.com/2346707/28708231-d540f09c-73be-11e7-9b60-5c4f8e333171.png">

[Also, this](https://www.youtube.com/watch?v=4R0oXjIzOx4).